### ISSUE TITLE
[bug] Bluetooth Media Reporting

### ISSUE THREAD

**Issue by attie** [leinelissen/jellyfin-audio-player#119]
I noticed that the track duration appears to be incorrectly reported to Bluetooth devices (e.g: car audio).

For example, listening to a `3:21` track, the car reports current position as `0:06` (fine), but remaining is shown as `-35791:17` (wrong, should be `-3:15` ish)

![image](https://user-images.githubusercontent.com/2195519/211869323-654a101d-3015-4d2d-8371-4d6b2e320488.png)

![image](https://user-images.githubusercontent.com/2195519/211870481-59108800-aa2f-449e-9046-bf6a63b263be.png)

**Comment by leinelissen** [comment]
Hey @attie, thanks for reporting this. Can you reliably reproduce this on other devices as well? If this is a problem, this is a bug in the native code of an upstream dependency (react-native-track-player). I can't find any similar problems on their repository, but it might be an undiscovered bug. Let me know!

**Comment by attie** [comment]
I don't think I have any other Bluetooth audio sinks that can show information like this, but I'll see what I can get access to and report back.

> If this is a problem, this is a bug in the native code of an upstream dependency (react-native-track-player).

Good to know, shall I report there instead?

**Comment by leinelissen** [comment]
> Good to know, shall I report there instead?

Yes, that would be best. If you can reproduce it, it's probably a bug in their KotlinAudio library. It's best to report here: https://github.com/doublesymmetry/react-native-track-player

Would love to hear back from you in case you find something!
